### PR TITLE
[Issue #81] Authorized routes for admins

### PR DESCRIPTION
**Pull request by tosinoni**
1. Implemented the bread crumbs for admin on navbar
2. fixed the navbar data-toggle issue
3. authorized the route for admins
4. changed homepage for admins to portal route 
5. changed homepage for normal users to chats

### CODE REVIEW

File: SECP-service/src/main/java/com/visucius/secp/resources/AdminResource.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private final AdminController adminController;
+
+    public AdminResource(AdminController adminController) {
+        this.adminController = adminController;
+    }
+
+    @POST
+    @Path("{id}")
+    @UnitOfWork
+    public Response registerAdmin(@Auth @PathParam("id") String id) {
```
Comment (mohameddahrouj) [inline SECP-service/src/main/java/com/visucius/secp/resources/AdminResource.java]: Nice

File: SECP-service/src/main/resources/assets/app/index.html [hunk 2]
```diff
@@ -59,6 +64,8 @@
         <script src="lib/sweetalert2/dist/sweetalert2.js"></script>
         <script src="lib/select2/dist/js/select2.js"></script>
         <script src="lib/moment/moment.js"></script>
+        <script src="lib/bootstrap-css/js/bootstrap.min.js"></script>
```
Comment (mohameddahrouj) [inline SECP-service/src/main/resources/assets/app/index.html]: I guess we just didn't have the proper bootstrap then

File: SECP-service/src/main/resources/assets/app/js/app.js [hunk 3]
```diff
@@ -45,43 +56,64 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date', 'rou
...
+            isAdmin: isAdmin
+          }
       })
       .when('/portal/manage/group', {
           templateUrl: 'views/portal/manage-group.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
```
Comment (mohameddahrouj) [inline SECP-service/src/main/resources/assets/app/js/app.js]: Nice, didn't know we can set the values here like that.

**Comment by mohameddahrouj** [comment]
Great work, I pulled your branch and tested everything.
Only thing I noticed is a little bug when a menu item from the navbar is clicked in mobile, the dropdown just persists even after an item has been clicked. I will fix it when merged.

File: SECP-service/src/main/java/com/visucius/secp/resources/AdminResource.java (hunk 1 above)
Comment (tosinoni) [reply SECP-service/src/main/java/com/visucius/secp/resources/AdminResource.java]: Yes lol. now we can register admin. I created 2 users with admin privileges.

File: SECP-service/src/main/resources/assets/app/index.html (hunk 2 above)
Comment (tosinoni) [reply SECP-service/src/main/resources/assets/app/index.html]: yep we forgot to add the bootstrap javascript file.

File: SECP-service/src/main/resources/assets/app/js/app.js (hunk 3 above)
Comment (tosinoni) [reply SECP-service/src/main/resources/assets/app/js/app.js]: yeah we can. The route wont be loaded unless that promise evaluates to true.

**Comment by tosinoni** [comment]
I fixed the persistence of navbar after a link is clicked, I totally missed that. Thanks.

**Pull request merged**